languages/python/omni_http/src/omni_http/omni_httpd/flask.py

```diff
@@ -27,12 +27,12 @@ class Adapter:
     def __init__(self, app: flask.Flask):
         self.app = app
 
-    def __call__(self, req: HTTPRequest) -> HTTPOutcome:
+    def __call__(self, req: HTTPRequest) -> HTTPResponse:
         res = responder()
         response = self.app.wsgi_app(req.wsgi_environ(), res)
 
         body = b""
         for chunk in response:
             body += chunk
 
-        return HTTPResponse(status = res.status, body = body, headers = res.headers).outcome()
\ No newline at end of file
+        return HTTPResponse(status = res.status, body = body, headers = res.headers)
```

Comment:
the python test script's `handle` function needs to be adjusted to:
`
@pg
def handle(req: omni_httpd.HTTPRequest) -> omni_httpd.HTTPOutcome:
    return app_(req).outcome()
`

Reply:
I think we can actually look into amending `omni_httpd` extension behavior.Right now it's looking for `http_outcome` only, but we can also let it look for `http_response` individually (if other responses are impossible; and may be allowed singling out other types of outcomes). This way no conversion would be necessary.